rollup.config.js

```diff
@@ -2,13 +2,21 @@
 
 var commonjs = require("rollup-plugin-commonjs");
 
-module.exports = {
-    input: "lib/index.js",
-    output: {
-        name: "sinonTest",
-        format: "umd"
-    },
-    plugins: [
-        commonjs({sourceMap: false})
-    ]
-};
+function setConfig(cfg) {
+    return {
+        input: "lib/index.js",
+        output: {
+            name: "sinonTest",
+            file: cfg.file,
+            format: cfg.format
+        },
+        plugins: [
+            commonjs({sourceMap: false})
+        ]
+    };
+}
+
+module.exports = [
+    setConfig({format: "umd", file: "dist/sinon-test.js"}),
```

Comment:
Mind renaming the `setConfig` to `createConfig`? `set*` functions usually have side effects, while this is a pure function.

Reply:
Otherwise, this is a fine improvement.

Reply:
Thanks, good point... Fixed...